src/containers/User/UserArea.tsx

```diff
@@ -312,7 +325,7 @@ const MainMenuBar: React.FC<MainMenuBarProps> = ({
             </Dropdown>
           </List.Item>
         )}
-        {intReferenceDocs.length && (
+        {intReferenceDocs.length > 0 && (
```

Comment:
I don't really why this worked before, but these were showing up as a "0" in the UI when the arrays were empty. 🤷‍♂️

Reply:
Yes, was having a bad day... Only had one language so option was not showing up in submenu of user options.